packages/console/src/console.js

```diff
@@ -1,8 +1,8 @@
 // @ts-check
 
-import { ErrorInfo } from '@agoric/assert';
+import { ErrorInfo } from '@agoric/assert/src/assert';
```

Comment:
why?

Reply:
The `@agoric/assert` import defaults to the CommonJS under Typescript (due to `assert/package.json` specifying `"main": "dist/assert.cjs"`.  This workaround to directly import the ESM was the only way I could find around it.